packages/evm/jsonrpc/evmchain.go

```diff
@@ -45,6 +45,8 @@ import (
 	"github.com/iotaledger/wasp/packages/vm/gas"
 )
 
+const maxLogsTopics = 4
```

Comment:
You recreate Go-Ethereums behavior here, right? Could you add a comment over the const to clarify this?